modules/ballerina-core/src/main/java/org/ballerinalang/util/parser/antlr4/BLangAntlr4Listener.java

```diff
@@ -2678,6 +2691,58 @@ public void exitXmlQualifiedName(XmlQualifiedNameContext ctx) {
     }
 
     @Override
+    public void enterStringTemplateLiteral(StringTemplateLiteralContext ctx) {
+
+    }
+
+    @Override
+    public void exitStringTemplateLiteral(StringTemplateLiteralContext ctx) {
+
+    }
+
+    @Override
+    public void enterStringTemplateContent(StringTemplateContentContext ctx) {
+
+    }
+
+    @Override
+    public void exitStringTemplateContent(StringTemplateContentContext ctx) {
+        if (ctx.exception != null) {
+            return;
+        }
+        List<TerminalNode> nodes = ctx.StringTemplateExpressionStart();
+        StringTemplateTextContext endTextNode = ctx.stringTemplateText();
+        String endingText = endTextNode == null ? null : StringEscapeUtils.unescapeJava(endTextNode.getText());
```

Comment:
Here unescape if the 'verboseMode' flag is not set. 

'verboseMode' flag is set by the composer to persist white spaces in the code.

Refer the 'createBasicLiteral' method in this class.